module-3/web-app/src/components/EditorContainer/EditorTitle/EditorTitle.jsx

```diff
@@ -2,10 +2,17 @@ import React from "react"
 import style from "./EditorTitle.module.scss"
 
 
-const EditorTitle = () => {
+const EditorTitle = ({onChange, documentTitle}) => {
+
   return (
-        <div><input type="text" name="title" placeholder="Enter your title here..."/></div>
+        <div><input
```

Comment:
Is there a real benefit of wrapping input into a div?

Reply:
Fixed